deps/media-playback/media-playback/media.h

```diff
@@ -41,8 +41,62 @@ extern "C" {
 typedef void (*mp_video_cb)(void *opaque, struct obs_source_frame *frame);
 typedef void (*mp_audio_cb)(void *opaque, struct obs_source_audio *audio);
 typedef void (*mp_stop_cb)(void *opaque);
+typedef bool (*mp_state_handler)(struct mp_media_t *m);
```

Comment:
This causes various warnings in VC, because `struct mp_media_t` is an unknown type.  Plus, mp_media_t is not a struct (technically it is but I'll get to that), it's a typedef. So you should not be specifying `struct mp_media_t`.

To fix this, just do a forward declare somewhere above this:
```C
struct mp_media;
typedef struct mp_media mp_media_t;
```
to be in accordance with the structure itself below, then change this line to:
```C
typedef bool (*mp_state_handler)(mp_media_t *m);
```